examples/moving_horizon_estimation/mhe_simulation.py

```diff
@@ -0,0 +1,81 @@
+import numpy as np
+import casadi as cas
+from scipy.integrate import solve_ivp
+import matplotlib.pyplot as plt
+
+
```

Comment:
What is the purpose of this file? If it is not to run any OCP, but to prepare things, can this be added to the relevant file only? If not, maybe renaming to mhe_utils.py?

Reply:
The purpose is  to simulate a set of noisy data, prior to running the mhe.